src/functionalTest/resources/features/F-205 - Validate NoC Questions/S-205.13.td.json

```diff
@@ -0,0 +1,45 @@
+{
+  "_guid_": "S-205.13",
+  "_extends_": "S-205_Test_Data_Base",
+  "specs" : [
+    "Richard - with the ability to create a case for a particular jurisdiction within an organisation",
+    "Dil - with a solicitor role for the same jurisdiction within a different organisation from Richard",
+    "is made by Dil to validate the answers provided to the NoC questions in order to become the representative for Mario on C1",
+    "contains answers that do not match a case role in the case to be represented"
+
+  ],
+
+  "users": {
+    "invokingUser": {
+      "_extends_": "ACA_Users_Dil"
+    }
+  },
```

Comment:
If two users in specs list should have two users here (with order matching the order in feature file).